sdk/ml/azure-ai-ml/tests/dsl/e2etests/test_controlflow_pipeline.py

```diff
@@ -256,11 +256,11 @@ def test_pipeline(input_data, int_param, bool_param, float_param, str_param):
         }
 
 
-@pytest.mark.skipif(
-    condition=is_live(),
-    # TODO: reopen live test when parallel_for deployed to canary
-    reason="parallel_for is not available in canary."
-)
+# @pytest.mark.skipif(
```

Comment:
directly remove these lines, rather than comment?

Reply:
Good catch, forgot to change this back